docker/with-external-db/with-db.env

```diff
@@ -5,19 +5,30 @@ APPLICATION_BASE_URL="http://localhost:3000"
 ALLOWED_ORIGINS="http://localhost:3000"
 
 # Database
-ENABLE_EXTERNAL_DB="false"
+ENABLE_EXTERNAL_DB="true"
+
+# Environment variables for external DB
+EXTERNAL_DB_CONNECTION_URL="postgres://<user>:<password>@<host>:<port>/<database>"
+EXTERNAL_DB_ENCRYPTION_KEY="<db-encryption-key>"
+EXTERNAL_DB_CERT="<path/to/extenal-db-cert>"
+
+# LogTo
+LOGTO_RESOURCE_URL="http://localhost:3000"
+LOGTO_ENDPOINT="http://localhost:3001"
+LOGTO_APP_ID="ldfsr...rq432"
+LOGTO_APP_SECRET="sdf...sdf"
 
 # Authentication
-ENABLE_AUTHENTICATION="false"
+ENABLE_AUTHENTICATION="true"
+
+# Environment variables for Authentication
+COOKIE_SECRET="sdf...sdf"
+ALL_SCOPES="account:create did:create credential:issue"
```

Comment:
Yup, I changed it in the latest PR with auth stuff and for now It asks for all the scopes using M2M token. I think, it could be removed while merging with latest develop

Reply:
Removed an unused `ALL_SCOPES` environment variable.